cedar-policy-validator/src/schema.rs

```diff
@@ -2913,9 +2913,14 @@ pub(crate) mod test {
         });
     }
 
-    // Names like `Set`, `Record`, `Entity`, and Extension` are not allowed as common type names, as specified in #1070.
+    // Names like `Set`, `Record`, `Entity`, and Extension` are not allowed as common type names, as specified in #1070 and #1139.
     #[test]
     fn test_common_type_name_conflicts() {
+        let assert_invalid_json_schema = |src: serde_json::Value| {
```

Comment:
This closure doesn't capture anything, so you make it a local `fn` definition with `#[track_caller]`

Reply:
Done.